frontend/src/main/utils/PersonalScheduleUtils.js

```diff
@@ -0,0 +1,17 @@
+import { toast } from "react-toastify";
+import { useNavigate } from 'react-router-dom'
+
+export function onDeleteSuccess(message) {
+    console.log(message);
+    toast(message);
+}
+
+export function cellToAxiosParamsDelete(cell) {
+    return {
+        url: "/api/personalschedules",
+        method: "DELETE",
+        params: {
+            id: cell.row.values.id
+        }
+    }
```

Comment:
Do you foresee that you will be using these in more than one location? If not, I'm curious as to why you decided to factor these functions into a separate util file instead of keeping this in the component directly.

Reply:
Fixed